changelog.d/.towncrier_template.md.jinja

```diff
@@ -0,0 +1,66 @@
+{%- set is_draft_preview = versiondata["version"] == "DRAFT_VERSION" -%}
+
+{%- if is_draft_preview -%}
+## Unreleased Changes
+{% else -%}
+## {{ versiondata["version"] }}
+{% endif %}
+
+*{%- if is_draft_preview -%}Generated for preview on {% endif -%}{{ versiondata["date"] }}*
+
+{% for section, _ in sections.items() %}
+{% if sections[section] %}
+{% for category, val in definitions.items() if category in sections[section] -%}
+
+{{ definitions[category]['name'] }}:
+
+{% for text, change_note_refs in sections[section][category].items() %}
+
+  {%-
+    set pr_issue_numbers = change_note_refs
+    | map('lower')
+    | map('int', default=None)
+    | select('integer')
+    | map('string')
+    | list
+  -%}
```

Comment:
I only now noticed that this doesn't have my hex replacement hack. Did some digging and realized that it's been fixed in Jinja at the end of last year: https://github.com/pallets/jinja/issues/1921.

Reply:
Oh, I forgot to call that out explicitly for you, sorry! Yeah, I traced it back and saw that it was fixed, so we didn't need it anymore. But I should have said something.

Reply:
Well, now I know :) Marking this as resolved.